src/_xoConnection.js

```diff
@@ -131,6 +137,34 @@ class XoConnection extends Xo {
     return id;
   }
 
+  async runBackupJob(
+    jobId,
+    { exportRetention, copyRetention },
+    schedule,
+    numberOfExecution
+  ) {
+    for (let i = 0; i < numberOfExecution; i++) {
+      await xo.call("backupNg.runJob", { id: jobId, schedule });
+    }
+    if (exportRetention || copyRetention) {
```

Comment:
It not delete backups it only delete replications. Please do it for backups we'll do it for replication when it will be needed.